core/src/main/scala/org/http4s/headers/Expires.scala

```diff
@@ -18,8 +13,14 @@ object Expires extends HeaderKey.Internal[Expires] with HeaderKey.Singleton {
   *
   * @param expirationDate the date of expiration
   */
-final case class Expires(expirationDate: HttpDate) extends Header.Parsed {
-  val key = `Expires`
-  override val value = Renderer.renderString(expirationDate)
-  override def renderValue(writer: Writer): writer.type = writer.append(value)
-}
+final case class Expires(expirationDate: java.time.Instant)
+object Expires {
+  implicit val expiresHeader: Header[Expires] =
+      new Header[Expires] {
+        val name = CaseInsensitiveString("Expires")
+        def parse(s: String) = HttpHeaderParser.EXPIRES(s).toOption
+        def toRawHeaders(a: Expires) = List(RawHeader(
+          name, Renderer.renderString(a.expirationDate)))
```

Comment:
Is this a good opportunity to phase out `Renderer`?

Reply:
Probably, same with the writer implementations.